core/App/fifo.cpp

```diff
@@ -47,7 +47,7 @@
 
 #define BUFFER_SIZE 1024
```

Comment:
A typo in the commit title
"add dkeyserver and ckeycache docker support" --> "add dkeyserver and dkeycache docker support"

Reply:
It will be fix